.TOOLVERSIONS

```diff
@@ -1,6 +1,6 @@
-android-ndk;r10e;070be287539e3e7706f8dabfb6bf9879
-android-sdk-build-tools;28.0.1;
-android-sdk-platform;android-27;
+android-ndk;r17c;a4b6b8281e7d101efd994d31e64af746
+android-sdk-build-tools;28.0.3;
+android-sdk-platform;android-28;
```

Comment:
btw it seems that `android-26` is hardcoded here https://github.com/status-im/status-react/blob/6d3211de46f302ab1a92eb3bb711733dd9cb8d9f/docker/android/Dockerfile#L13

Reply:
I'm not too familiar with the Android SDK dependencies, I assume that we need this baseline 26 platform for some reason.

Reply:
Yeah, it is, that's kinda how `sdkmanager` works, the packages included are named after the version of sdk. I guess it could be parametrized.

Reply:
Ah wait, it actually already is parametrized right below it:
https://github.com/status-im/status-react/blob/6d3211de46f302ab1a92eb3bb711733dd9cb8d9f/docker/android/Dockerfile#L14
So my guess would be that when I was building this it wouldn't work if also `platform:android-26` was not installed.